src/sales/order-workflow.md

```diff
@@ -8,13 +8,13 @@ Step 1: Place Order
 : The checkout process begins when the shopper clicks the Go to Checkout button on the shopping cart page or [reorders]({% link sales/reorders-allow.md %}) directly from their customer account.
 
 Step 2: Order Pending
-: In the Orders grid, the status of the sales order is initially `Pending`. Payment has not been processed, and the order can still be edited or canceled.
+: The status of the sales order is initially `Pending`. Payment has not been processed, and the order can still be edited or canceled. This will occur when payment processor module is configured for authorization mode.
```

Comment:
This will occur -- not sure what the last sentence means. What does "This" refer to?   Are you saying that an order only goes into Pending status for Payment methods that have been configured with the "authorization" action?  (See [Payment Methods - Payment actions]((https://docs.magento.com/user-guide/configuration/sales/payment-methods.html#payment-actions).

From the Adobe Commerce end-user perspective,  "payment processor module" the same as the Payment Method.